src/crypto/crypto_ec.cc

```diff
@@ -372,13 +372,13 @@ bool ECDH::IsKeyValidForCurve(const BignumPointer& private_key) {
   CHECK(private_key);
   // Private keys must be in the range [1, n-1].
   // Ref: Section 3.2.1 - http://www.secg.org/sec1-v2.pdf
-  if (BN_cmp(private_key.get(), BN_value_one()) < 0) {
+  if (private_key < BignumPointer::one()) {
```

Comment:
if this is a constant, should we make it all capital?

Reply:
I would if it were a regular constant but given that this calls down to an openssl function I'd rather keep it generally as is.... with the exception of upper-casing the `O`

Reply:
Yup agreed